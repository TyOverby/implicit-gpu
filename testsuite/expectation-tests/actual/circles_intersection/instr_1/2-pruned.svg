version https://git-lfs.github.com/spec/v1
oid sha256:d35cedd384a761487dbab1992123212b6bc29e295690b4c346756800f63dbb43
size 4945
