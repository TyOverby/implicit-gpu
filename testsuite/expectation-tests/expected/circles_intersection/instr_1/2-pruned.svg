version https://git-lfs.github.com/spec/v1
oid sha256:e0e67e655479396890bcbd40bf85d124e51f26405425b9bb5c135697441e59ef
size 4853
