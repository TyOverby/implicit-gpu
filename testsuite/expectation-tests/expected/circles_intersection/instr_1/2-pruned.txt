32 line segments

Line Segment 0 
2 points
Clockwise? false
(15.0,20.164022)
(14.0,20.538208)

Line Segment 1 
2 points
Clockwise? false
(14.0,20.538208)
(13.0,20.79761)

Line Segment 2 
2 points
Clockwise? false
(12.0,20.949848)
(11.050152,20.0)

Line Segment 3 
2 points
Clockwise? true
(11.835978,17.0)
(12.0,16.661701)

Line Segment 4 
2 points
Clockwise? false
(20.538208,14.0)
(20.164022,15.0)

Line Segment 5 
2 points
Clockwise? true
(14.0,13.862445)
(15.0,13.0)

Line Segment 6 
2 points
Clockwise? false
(18.0,18.137556)
(17.0,19.0)

Line Segment 7 
2 points
Clockwise? true
(11.461792,18.0)
(11.835978,17.0)

Line Segment 8 
2 points
Clockwise? true
(20.0,11.050152)
(20.949848,12.0)

Line Segment 9 
2 points
Clockwise? false
(19.0,17.0)
(18.137556,18.0)

Line Segment 10 
2 points
Clockwise? true
(17.0,11.835978)
(18.0,11.461792)

Line Segment 11 
2 points
Clockwise? false
(16.0,19.656902)
(15.338299,20.0)

Line Segment 12 
2 points
Clockwise? false
(13.0,20.79761)
(12.0,20.949848)

Line Segment 13 
2 points
Clockwise? false
(20.0,15.338299)
(19.656902,16.0)

Line Segment 14 
2 points
Clockwise? true
(11.050152,20.0)
(11.202392,19.0)

Line Segment 15 
2 points
Clockwise? false
(17.0,19.0)
(16.0,19.656902)

Line Segment 16 
2 points
Clockwise? true
(16.0,12.343098)
(16.661701,12.0)

Line Segment 17 
2 points
Clockwise? true
(13.862445,14.0)
(14.0,13.862445)

Line Segment 18 
2 points
Clockwise? true
(13.0,15.0)
(13.862445,14.0)

Line Segment 19 
2 points
Clockwise? true
(11.202392,19.0)
(11.461792,18.0)

Line Segment 20 
2 points
Clockwise? false
(15.338299,20.0)
(15.0,20.164022)

Line Segment 21 
2 points
Clockwise? false
(19.656902,16.0)
(19.0,17.0)

Line Segment 22 
2 points
Clockwise? true
(12.343098,16.0)
(13.0,15.0)

Line Segment 23 
2 points
Clockwise? true
(16.661701,12.0)
(17.0,11.835978)

Line Segment 24 
2 points
Clockwise? false
(20.79761,13.0)
(20.538208,14.0)

Line Segment 25 
2 points
Clockwise? false
(20.164022,15.0)
(20.0,15.338299)

Line Segment 26 
2 points
Clockwise? false
(18.137556,18.0)
(18.0,18.137556)

Line Segment 27 
2 points
Clockwise? true
(18.0,11.461792)
(19.0,11.202392)

Line Segment 28 
2 points
Clockwise? true
(15.0,13.0)
(16.0,12.343098)

Line Segment 29 
2 points
Clockwise? false
(20.949848,12.0)
(20.79761,13.0)

Line Segment 30 
2 points
Clockwise? true
(12.0,16.661701)
(12.343098,16.0)

Line Segment 31 
2 points
Clockwise? true
(19.0,11.202392)
(20.0,11.050152)
